client/components/app.jsx

```diff
@@ -95,6 +95,7 @@ export default class App extends React.Component {
 
   componentDidMount() {
     this.getCartItems();
+    alert('This website is a demo and no real purchases will be made!');
```

Comment:
`alert`s can be easily bypassed, since there is a browser option to ignore them. I'd suggest using an in-page component (a modal or some other type of dialog) instead.